app/main.py

```diff
@@ -98,6 +98,7 @@ class Enemy:
     def __init__(self, prepend):
         self.coord = [[prepend['body']['data'][j]['y'], prepend['body']['data'][j]['x']] for j in range(len(prepend['body']['data']))]
         self.length = prepend['length']
+        self.tail = prepend['body(length-1)'] #should just be the last coordinates (point) in the list (body) - Im assuming this is the tail. But my syntax is not great, is this correct? 
```

Comment:
Correct syntax would be `self.tail = [prepend['body']['data'][-1]['y'], prepend['body']['data'][-1]['x']]`. Anyway, this also got me thinking about adding a head property too so that it can avoid the tail if the head is one step behind a food, since the tail will remain meaning the snake would run into the tail.

Reply:
Awesome, thanks for the tip on syntax! I was wondering how that would look. 
I was also thinking about the issue of the snake ahead of ours eating. My solution was to follow 2 blocks behind, but I like your idea better - that way we can change directions if the snake ahead of us is about to eat an apple, and we have the added protection of being that much closer to a tail.